tox.ini

```diff
@@ -46,28 +43,22 @@ commands =
     pip install pywlroots>=0.16.4,<0.17.0
     pip install xcffib>=1.4.0 wheel
     pip install cairocffi>=1.6.0
-    pip install -v git+https://github.com/elParaguayo/qtile.git@fixing-packaging
+    pip install git+https://github.com/elParaguayo/qtile.git@fixing-packaging
     ; pip install -e git+https://github.com/qtile/qtile.git#egg=qtile
-    python3 setup.py install
+    pip install .
```

Comment:
It doesn't look like the docs reflect this

Reply:
They do, just not in the default, "stable" page. See https://qtile-extras.readthedocs.io/en/latest/manual/install.html#everyone-else